app/src/main/java/com/github/geohunt/app/model/database/api/Location.kt

```diff
@@ -74,6 +86,31 @@ data class Location(var latitude: Double = 0.0,
         return "$degree° $minutes' ${String.format("%.2f", value)}\"${if (v > 0.0) positive else negative}"
     }
 
+    companion object {
+        /**
+         * Mean earth radius in kilometers, used to compute the distance between two Locations
+         */
+        private const val EARTH_MEAN_RADIUS = 6371
+        
+         /**
+         * Length of the string representing the coarse hash
+         */
+        const val COARSE_HASH_SIZE = (2 * Long.SIZE_BYTES) / 2;
```

Comment:
This constants could be made public, as it could be useful to users of the class